yarn.lock

```diff
@@ -2854,6 +2854,14 @@ bitcoin-ops@^1.3.0:
   dependencies:
     nanoassert "^1.0.0"
 
+"blake2b@git+https://github.com/BitGo/blake2b.git#6268e6dd678661e0acc4359e9171b97eb1ebf8ac":
+  version "2.1.3"
+  uid "6268e6dd678661e0acc4359e9171b97eb1ebf8ac"
+  resolved "git+https://github.com/BitGo/blake2b.git#6268e6dd678661e0acc4359e9171b97eb1ebf8ac"
+  dependencies:
+    blake2b-wasm "https://github.com/BitGo/blake2b-wasm#193cdb71656c1a6c7f89b05d0327bb9b758d071b"
+    nanoassert "^1.0.0"
+
```

Comment:
Apart from the `uid` lines, these look exactly the same as the following ones.

More importantly, if I understand correctly, this is an implementation of `BLAKE2b` as part of [BLAKE2](https://blake2.net/). Decred uses `BLAKE-256` as part of the original [BLAKE](https://131002.net/blake/), which is a different algorithm.